frontend/express/public/stylesheets/main.css

```diff
@@ -2841,4 +2832,380 @@ color:#E95C6C; transition:color 1s;
+    display: block;
+    width: calc(100% - 14px);
+    z-index: auto;
+    position: relative;
+    line-height: normal;
+    font-size: 11px;
+    transition: none 0s ease 0s;
+    background: #fff !important;
+    resize: none;
+    font-size:14px;
+    padding-left: 10px;
+}
+
+.graph-note-create .button {
+    visibility: visible;
+    cursor: pointer;
+    user-select: none;
+    background-color: #ececec;
+    float: left;
+    border: 1px solid #d0d0d0;
+    margin-right: -1px;
+    color: #636363;
+    width: 68px;
+    height: 30px;
+    text-align: center;
+    width: 32.8%;
+    line-height: 30px;
+    font-size: 13px;
+}
+.graph-note-create .note-types-selector .button:nth-child(2) {
+    border-top-left-radius: 2px;
+    border-bottom-left-radius: 2px;
+}
+.graph-note-create .button:last-child {
+    border-top-right-radius: 2px;
+    border-bottom-right-radius: 2px;
+    border-right: 1px solid #d0d0d0;
+}
+.graph-note-create .calendar {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+.graph-note-create .date-time-selector-container i {
+    font-size: 14px;
+    margin: 5px 5px;
+}
+.graph-note-create .date-time-selector-container .date-time-value-show {
+    flex-grow: 1;
+    padding-left: 10px;
+}
+.graph-note-create .button.active {
+    background-color: #ffffff;
+}
+.graph-note-create .selectize-control.multi .selectize-input > div {
+    padding: 3px 22px 3px 10px !important;
+    line-height: 14px !important;
+    background-color: #777;
+    color: #f1f1f1;
+}
+.note-table-view-navigator .back-link {
+    cursor: pointer;
+    margin-bottom: 25px;
+    color: #616161;
+    font-size: 12px;
+    display: inline-block;
+    text-transform: uppercase;
+}
+.note-table-view-navigator .back-link:before {
+    content: "←";
+    margin-right: 5px;
+    color: #616161;
+}
+
+.date-time-selector-container {
+    width: 90%;
+    border: 1px solid #C5C5C5;
+    background-color: #fff;
+    color: #636363;
+    font-size: 14px;
+    line-height: 30px;
+    height: 30px;
+    border-radius: 2px;
+}
+.date-time-selector-container .time-field {
+    padding: 10px;
+    font-size: 13px;
+    display: flex;
+    align-items: center;
+}
+.date-time-selector-container .time-label {
+    color: #fff;
+    flex-grow: 1;
+}
+.date-time-selector-container .time-split {
+    color: #fff;
+}
+.date-time-selector-container .time-input {
+    width: 40px;
+    height: 24px;
+    background-color: #3d3d3d;
+    border: none;
+    outline: 0;
+    font-size: 13px;
+    color: #fff;
+    text-align: center;
+    border-radius: 2px;
+    line-height: 2;
+}
+.date-time-selector-container .date-time-picker {
+    display: none;
+    white-space: nowrap;
+    display: none;
+    position: absolute;
+    z-index: 1000;
+    padding: 2px 3px 6px 3px;
+    background-color: #292929;
+    border-radius: 2px;
+    top: -25px;
+    left: 102px;
+    flex-direction: column-reverse;
+}
+.tipsy-inner {max-width: 400px !important; }
```

Comment:
Similar here, changing all tipsy styles would affect them in all places, not just notes

Reply:
solved